node/core/chain-selection/src/lib.rs

```diff
@@ -398,6 +426,23 @@ where
 							).await?;
 
 							backend.write(write_ops)?;
+
+							if is_validator != Validator::new(leaf.hash, keystore.clone(), &mut sender).await.is_ok() {
+								is_validator = !is_validator;
+								tracing::info!(
+									"👮 Authority status changed to `{}` at block #{}({})",
```

Comment:
This is also wrong on node startup, because the authority status didn't changed. Just your boolean changes here.

Reply:
This is as intended - it confirms the node is in the authority set. The semantics are off indeed when the node starts, but for debugging purposes this still has value.

Reply:
Nodes don't restart that often normally (assuming they are not crashing because of a bug), so you will need to search in loki for this piece of log line. 

I still don't see the advantage. With a prometheus metric you could at least see the value directly.

Reply:
IMO it's reasonable to search the logs for this piece of information, but I agree that metrics would convey the same information more efficiently. I am a bit on the fence, but it seems like consensus on this PR is that this should be a metric.